next/app/projects/page.tsx

```diff
@@ -21,15 +21,15 @@ export default async function ProjectsPage() {
       <section className="h-[85vh] flex flex-col items-center justify-center relative">
         <Image
           src={getLargestImageUrl(projectsPage.BackgroundImage)}
-          alt=""
-          className="absolute inset-0 -z-10 object-cover w-full h-full brightness-50"
+          alt="Projects page background image"
+          className="absolute inset-0 -z-10 object-cover object-[center_bottom] w-full h-full brightness-50"
```

Comment:
P.s.! Is there a reason you chose to bottom-align images? Not right or wrong just curious~~